content/blog/2019-02-01-software-peer-review-news.md

```diff
@@ -0,0 +1,80 @@
+
+Since our [last post about updates in the system in September 2017](/blog/2017/09/11/software-review-update/), the editorial team has doubled in size to handle all incoming submissions without drowning editors! We are delighted to have welcomed Anna Krystalli, Lincoln Mullen, Brooke Anderson and Melina Vidoni. Thanks to them for joining us! Read more [about Anna and Lincoln](/blog/2018/06/22/new_editors/) and about Brooke and Melina (soonish?).
+
+### We wrote a whole book/manual!
+
+{{< figure src="/img/blog-images/2019-02-01-software-review-news/guide.png" alt="Screenshot from our online book" width="700" >}}
+
+Our guidance for authors, reviewers and editors used to be scattered over single Markdown files in the Software Peer Review GitHub repository, which was getting impractical as these files got bigger, and which was not giving a great reading experience to anyone. We have therefore consolidated all our guidance to [a book](https://ropensci.github.io/dev_guide/) created with `bookdown`! Its source is [stored on GitHub](https://github.com/ropensci/dev_guide). It is divided in three sections: 
+
+* [one about our guidelines and tips for package development](https://ropensci.github.io/dev_guide/building.html), useful to read for any package developer.
+
+* [one about the Software Peer Review system itself](https://ropensci.github.io/dev_guide/softwarereviewintro.html), including guidance for the different actors of the system.
+
+* [one about package maintenance](https://ropensci.github.io/dev_guide/collaboration.html), including a chapter about changing stuff in your package, one about GitHub grooming... This section too can be useful for any package developer!
+
+This blog post marks the official release of our guide, whose updates are tracked [via a NEWS appendix](https://ropensci.github.io/dev_guide/booknews.html). The next big update won't be about packaging guidelines though: we're working on adding a legit cover to the online book with Locke Creatives, stay tuned!
+
+### Other smaller updates
+
+We've made a bunch of other smaller changes.
+
+#### Issue templates
+
+{{< figure src="/img/blog-images/2019-02-01-software-review-news/issue_templates.png" alt="Screenshot of issue template selection" width="700" >}}
+
+When [clicking on "new issue" in the Software Peer Review Github repository](https://github.com/ropensci/software-review/issues/new/choose), one gets to choose between submission, pre-submission and feature request/bug report.
+
+{{< figure src="/img/blog-images/2019-02-01-software-review-news/submission.png" alt="Screenshot of submission issue template" width="700" >}}
+
+The first two reasons for opening an issue have their own issue template, and the last one guides people towards [our repository aimed at discussion of the system](https://github.com/ropensci/software-review-meta/). We hope it makes it easier to participate.
+
+#### Changes in our standards
```

Comment:
This should be higher in the post - I think it's the most important aart.  I also think there like there should be more here.  Maybe look at the blame log for this? It should definitely include the expansion of scope to include system wrappers, as that should be approved before this post is out, too.
https://github.com/ropensci/dev_guide/blame/master/pkg_building.Rmd
https://github.com/ropensci/dev_guide/blame/master/softwarereview_policies.Rmd

Reply:
I can't remember how I made the list in https://github.com/ropensci/software-review-meta/issues/72 but I'm pretty sure it was by looking at different git histories and GitHub closed issues, will look again to see if I find other missing things.

Reply:
Added everything that I've found, indeed a few items had been missed :scream: